functions_for_gyre.py

```diff
@@ -41,50 +42,100 @@ def calc_scanning_range(gyre_file_path, npg_min=-50, npg_max=-1, l=1, m=1, omega
     l, m: integer
         degree (l) and azimuthal order (m) of the modes
     omega_rot: float
-        rotation frequency of the model [c/d]
+        rotation frequency of the model (c/d) 
 
     ------- Returns -------
     f_min, f_max: float
         lower and upper bound of frequency range that needs to be scanned in oder
         to retrieve the required range of radial orders
     """
-    directory, gyre_file = mypy.split_line(gyre_file_path, 'gyre/')
-    Xc_file = float(mypy.substring(gyre_file, 'Xc', '.GYRE'))
-    hist_file = glob.glob(f'{directory}*.hist')[0]
-    dic_hist  = read.read_multiple_mesa_files([hist_file], is_hist=True, is_prof=False)[0]
-    data      = dic_hist['hist']
+    directory, gyre_file = mypy.split_line(gyre_file_path, 'gyre/') # get directory name and GYRE filename
+    Xc_file = float(mypy.substring(gyre_file, 'Xc', '.GYRE')) # get Xc
+    hist_file = glob.glob(f'{directory}*.hist')[0] # selects the first history file in the folder
+    dic_hist  = read.read_multiple_mesa_files([hist_file], is_hist=True, is_prof=False)[0] # read the MESA files into a dictionary
+    # Retrieve data
+    data      = dic_hist['hist'] 
     P0_values = data['Asymptotic_dP']
     Xc_values = data['center_h1']
 
-    # get the asymptotic period spacing value at the Xc value of the gyre file
+    # Obtain the asymptotic period spacing value/buoyancy radius at the Xc value closest to that of the gyre file
     diff = abs(Xc_file - Xc_values)
     xc_index = np.where(diff == np.min(diff))[0]
-    P0 = P0_values[xc_index][0]
+    P0 = P0_values[xc_index][0] # asymptotic period spacing value/buoyancy radius
 
-    # calculate the scanning range a bit broader than the purely asymptotic values, just to be safe.
+    # Calculate the scanning range a bit broader than the purely asymptotic values, just to be safe.
     n_max_used = abs(npg_min-3)
     n_min_used = abs(min(-1, npg_max+3))
 
     if omega_rot==0:
+        # If no rotation, use asymptotic values
         f_min = np.sqrt(l*(l+1)) / (n_max_used*P0)
         f_max = np.sqrt(l*(l+1)) / (n_min_used*P0)
     else:
-        # Make a pandas dataframe
-        df = pandas.read_csv(os.path.expandvars('$CONDA_PREFIX/lib/python3.7/site-packages/PyPulse/lambda.csv'), sep=',')
-        # select nu (Spin parameter) and Lambda (eigenvalues) column when values in l and m column correspond to requested values
+        # Make a pandas dataframe containing an interpolation table for lambda (eigenvalues of LTE - TAR)
+        df = pandas.read_csv(os.path.expandvars('$CONDA_PREFIX/lib/python3.7/site-packages/mamsie_py/lambda.csv'), sep=',')
```

Comment:
'mamsie_py' should be replaced by 'PyPulse'

Reply:
`         df = pandas.read_csv(os.path.expandvars('$CONDA_PREFIX/lib/python3.7/site-packages/PyPulse/lambda.csv'), sep=',')`